### PR TITLE
add triangular_solve/poisson/bitwise_and/or/xor/not OP benchmark

### PR DESCRIPTION
**Pull request by zhouwei25**
triangular_solve
---
triangular_solve

poisson
---
poisson

bitwise and/or/xor
---
infoflow 2022-03-03 19-35-42

bitwise_not
---
bitwise_not

### CODE REVIEW

File: api/dynamic_tests_v2/bitwise_not.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# limitations under the License.
+
+from common_import import *
+
+
+class BitwiseNotConfig(APIConfig):
+    def __init__(self):
+        super(BitwiseNotConfig, self).__init__('bitwise_not')
+        self.feed_spec = [{"range": [-100, 100]}, {"range": [-100, 100]}]
+        # abs belongs to activation op series which only has one parameter
```
Comment (ZzSean) [inline api/dynamic_tests_v2/bitwise_not.py:22]: 这里注释稍微改下

File: api/dynamic_tests_v2/bitwise_not.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```
Comment (ZzSean) [inline api/dynamic_tests_v2/bitwise_not.py:1]: copyright改下

File: api/dynamic_tests_v2/poisson.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
```
Comment (ZzSean) [inline api/dynamic_tests_v2/poisson.py:1]: 同上

File: api/dynamic_tests_v2/bitwise_not.py (hunk 1 above)
Comment (zhouwei25) [reply api/dynamic_tests_v2/bitwise_not.py]: Done, thx

File: api/dynamic_tests_v2/bitwise_not.py (hunk 2 above)
Comment (zhouwei25) [reply api/dynamic_tests_v2/bitwise_not.py]: Done, thx

File: api/dynamic_tests_v2/poisson.py (hunk 3 above)
Comment (zhouwei25) [reply api/dynamic_tests_v2/poisson.py]: Done, thx

**Review by ZzSean — APPROVED** [review]
LGTM

**Pull request merged**